### ISSUE TITLE
forecasting time siries

### ISSUE THREAD

**Issue by JGmancuso** [JGmancuso/deep-learning-test#1]
Hi.

I'm traying to forecast prices stock with Tensor Flow, first I'm an amateur.

My model is:
```
model = Sequential()
model.add(TimeDistributed(Conv1D(filters=64,kernel_size=1,
                                 activation='relu'), input_shape=(None, n_steps, n_features)))
model.add(TimeDistributed(Conv1D(filters=64, kernel_size=1, activation='relu')))
model.add(TimeDistributed(MaxPooling1D(pool_size=2)))
model.add(TimeDistributed(Flatten()))
model.add(LSTM(50, activation='relu',return_sequences=True)) # we do no normalization, so use relu
model.add(LSTM(10, activation='relu'))
model.add(Dense(1))
model.compile(optimizer='adam', loss='mse')
```
My final loss is 0.5 I think it's pretty good for a non standarize siries, but when I test the predictions with the real quotes, I saw delay model, my model never it's foward. I now that predict the exact prices its's impossible but I yes expect forecast the real movment.

Any idea that how can I forecast rightly?

![image](https://user-images.githubusercontent.com/61671037/75638583-534a5f00-5c0b-11ea-9676-178a0a2fe93b.png)

**Comment by ChaitanyaBaweja** [comment]
Did you try using a standard multivariate LSTM for this purpose rather than the CNN-LSTM model? I am not entirely sure but I fell that this could be due to the feature loss in early conv layers. You can find a good guide for LSTM for time series forecasting here: https://machinelearningmastery.com/multivariate-time-series-forecasting-lstms-keras/. Kindly share your results.
